mungegithub/mungers/publisher.go

```diff
@@ -354,7 +422,7 @@ func (p *PublisherMunger) publish() error {
 		for _, branchRule := range repoRules.srcToDst {
```

Comment:
If you add this, it's possible to test the script in dry run, skipping only the "git push" at the end:

		if p.githubConfig.DryRun {
			continue
		}